mempool/src/tests/shared_mempool_test.rs

```diff
@@ -237,7 +240,12 @@ impl SharedMempoolNetwork {
         let full_node = PeerId::random();
 
         // validator config
-        let mut config = NodeConfig::random();
+        let mut rng = StdRng::from_seed([0u8; 32]);
```

Comment:
I don't believe this is necessary and it conflicts with how we've setup other networks (to some extent).

Reply:
this is actually what the `NodeConfig::random()` does, I just make sure that the two calls in this function produce two different node configs instead of the same one.

Reply:
I mean I don't think this change is necessary -- was this test failing?

Reply:
Yeah this test is flaky, see https://github.com/diem/diem/pull/6826#issuecomment-749220605

I'll leave it up to you if you think it's not necessary :) I saw that as an easy fix of something that didn't look right.